packages/css-data/bin/property-value-descriptions.ts

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable no-console, func-style */
+import fetch from "isomorphic-fetch";
+import * as fs from "node:fs";
+import * as path from "node:path";
+import { keywordValues } from "../src/__generated__/keyword-values";
+
+/**
+ * Using ChatGPT, this scripts generates descriptions for CSS properties and declarations (property-value).
+ * It uses `keywordValues` to get a list of all the properties and values.
+ *
+ * The result is a TS module located at ./src/__generated__/property-value-descriptions.ts which exports:
+ * - properties
+ * - declarations
+ *
+ * By default it generates descriptions only for new properties/values, however you can use the --forceRegenerate flag to regenerate everything.
+ *
+ * When it comes to generating descriptions for declarations whose value is a color eg. `background-color: orange`
+ * we generate a description for a generic `property: {color}` declaration (e.g. `background-color: {color}`)
+ * and GPT will generate a description template with a placeholder `{color}`.
+ * The script repeats the template for every color value, replacing `{color}`.
+ * This is to avoid generating a ton of similar description and wasting GPT tokens.
+ *
+ * The script needs two env variables:
+ * - process.env.OPENAI_KEY - Your OpenAI API key https://platform.openai.com/account/api-keys
+ * - process.env.OPENAI_ORG - Your OpenAI Org ID https://platform.openai.com/account/org-settings
+ */
+
+const args = process.argv.slice(2);
+let forceRegenerate = false;
+if (args.includes("--forceRegenerate")) {
+  forceRegenerate = true;
+  args.splice(args.indexOf("--forceRegenerate"), 1);
+}
+
+const fileName = "property-value-descriptions.ts";
+const targetPath = path.join(process.cwd(), "src", "__generated__");
+
+let propertiesDescriptions: Record<string, string> = {};
+let declarationsDescriptions: Record<string, string> = {};
+
+try {
+  ({
+    properties: propertiesDescriptions,
+    declarations: declarationsDescriptions,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore Fix this else it'll complain that we cannot use top-level await.
+  } = await import(path.join(targetPath, fileName)));
+  if (!propertiesDescriptions) {
```

Comment:
we are having an internal rule, which may sound weird, but we are not using !before word, because in many fonts its unreadable `!i`, we use if (propertiesDescriptions === false) for example